README.md

```diff
@@ -66,6 +66,29 @@ Pycytominer is primarily built on top of [pandas](https://pandas.pydata.org/docs
 
 Pycytominer currently supports [parquet](https://parquet.apache.org/) and compressed text file (e.g. `.csv.gz`) i/o.
 
+### CellProfiler support
+
+Currently, Pycytominer fully supports data generated by [CellProfiler]('https://cellprofiler.org/'), adhering defaults to its specific data structure and naming conventions.
+
+CellProfiler-generated image-based profiles typically consist of two main components:
+
+- **Metadata features:** This section contains information about the experiment, such as plate ID, well position, incubation time, perturbation type, and other relevant experimental details. These feature names are prefixed with `Metadata_`, indicating that the data in these columns contain metadata information.
+- **Morphology features:** These are the quantified morphological features prefixed with the default compartments (`Cells_`, `Cytoplasm_`, and `Nuclei_`). Pycytominer also supports non-default compartment names.
```

Comment:
Is there an example of a non-default compartment name we could show here?

Reply:
I don't have experience working with non-default settings, but one potential issue could be changes in compartment names, such as differences in capitalization. In such cases, users would need to use the compartments argument as shown [here](https://github.com/cytomining/pycytominer/blob/4b772a5d7e7ded6c5ae67a024f28102c4d2305db/pycytominer/cyto_utils/util.py#L18). This adjustment would also affect the linking of columns as well [here](https://github.com/cytomining/pycytominer/blob/4b772a5d7e7ded6c5ae67a024f28102c4d2305db/pycytominer/cyto_utils/single_cell_ingest_utils.py#L5).

Reply:
i think @d33bs is just asking for simple example, which I like the idea of. I'll make the change